.github/workflows/trackdeploys.yaml

```diff
@@ -0,0 +1,55 @@
+name: Track semVer through environments
+
+on:
+  schedule:
+    - cron: '0 */3 * * *' # run every 3 hours and on push
+  workflow_dispatch: {}
+
+jobs:
+  get_versions:
+    name: Get semVer for environments
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_BUCKET_SA_KEY }}
+          export_default_credentials: true
+      - name: Use gcloud copy endpoints.json
+        run: gsutil cp gs://deployment-tracking/endpoints.json .
+      - shell: bash
+        run: |
+          endPoints+=( "https://jade.datarepo-dev.broadinstitute.org/configuration" \
+            "https://data.alpha.envs-terra.bio/configuration" \
+            "https://data.staging.envs-terra.bio/configuration" \
+            "https://data.terra.bio/configuration")
+          date=$(date +%Y-%m-%d\ %H:%M:%S)
+
+          for i in ${endPoints[@]}; do
+            if [[ "$i" == *"dev"* ]]
```

Comment:
Ah man I wish we could use objects here...

Reply:
Actually, could this just be in an endpoints.json so we didn't have to do bash logic?

Reply:
Switched to  jq to ditch those ifs